actionview/lib/action_view/lookup_context.rb

```diff
@@ -67,7 +67,7 @@ class DetailsKey #:nodoc:
       def self.get(details)
         if details[:formats]
           details = details.dup
-          details[:formats] &= Mime::SET.symbols
+          details[:formats] &= Template::Types::Type.types.to_a
```

Comment:
Also won't get enriched when Action Dispatch is required. So I'm adding a `Template::Types.symbols` method to paper over those differences in https://github.com/rails/rails/pull/23089